cmd/controller/app/server.go

```diff
@@ -113,6 +115,15 @@ func Run(o *options.ControllerRunOptions, ctx context.Context) error {
 	clusterInformers := clusterv1informers.NewSharedInformerFactory(clusterClient, 10*time.Minute)
 	kubeInfomers := kubeinformers.NewSharedInformerFactory(kubeClient, 10*time.Minute)
 
+	clusterRoleBindingsInformer := kubeInfomers.Rbac().V1().ClusterRoleBindings()
+	clusterRolesInformer := kubeInfomers.Rbac().V1().ClusterRoles()
+	roleBindingsInformer := kubeInfomers.Rbac().V1().RoleBindings()
+
+	//Init the used kube Informer, if we do not call it expicity, the related informer may not run.
+	clusterRoleBindingsInformer.Informer()
```

Comment:
I do not fully understand, you have called this in the sync controller.

Reply:
Previously, we only call the `rolesBindingInformer.Informer().HasSynced` in waitforcachesync func, which is after the kube informer start. 
So when the kubeinformer start, the rolebinding informer will not start.